tests/testthat/_snaps/linux-4.4/fsa/test-fsa-002.json

```diff
@@ -320,7 +320,7 @@
         "Your HMIS CSV Export should contain - with identical, case-sensitive spelling - only the columns specified in the columns.csv file. Please remove any extra columns and make sure all columns represented in the HMIS CSV Format Specifications are present.",
         "Your HMIS CSV Export should contain - with identical, case-sensitive spelling - only the columns specified in the columns.csv file. Please remove any extra columns and make sure all columns represented in the HMIS CSV Format Specifications are present.",
         "Per the HMIS CSV Format Specifications, all PersonalIDs in the Enrollment file should have a record in the Client file.",
-        "Per the HMIS CSV Format Specifications, all ProjectIDs in the Enrollment file should have a record in the Project file.",
+        "Per the HMIS CSV Format Specifications, all ProjectIDs in the Enrollment file should have a record in the Proect file.",
```

Comment:
(again, and I'm seeing in a few other places)

Reply:
fixed in EvaChecks.csv